src/global-event/index copy.tsx

```diff
@@ -0,0 +1,125 @@
+import global from '@dojo/framework/shim/global';
```

Comment:
Don't think this file was intentional!

Reply:
Yeah, not intentional. Thanks for catching this!